gradle.properties

```diff
@@ -1,3 +1,7 @@
 version=1.0-SNAPSHOT
 android.useAndroidX=true
-android.enableJetifier=true
\ No newline at end of file
+android.enableJetifier=true
+
+org.gradle.caching=true
```

Comment:
Thanks for the improvements : ] Those properties should speed up the builds a little bit